src/Engine/ProtoCore/Utils/StringUtils.cs

```diff
@@ -8,6 +8,7 @@ namespace ProtoCore.Utils
 {
     public static class StringUtils
     {
+        //TODO consider removing this option. It makes sharing harder.
```

Comment:
This sort of looks similar to the scale factor setting issue where we have one in preferences and another setting in the workspace (graph), where the latter overrides the former. I would have suggested adding this format specifier at the graph level as well but I don't think that's needed if we're going to deprecate the old string nodes at some point.